krun/results.py

```diff
@@ -11,7 +11,14 @@ class Results(object):
     Can be serialised to disk.
     """
 
+    # We use this to detect times where a results instance is loaded prior to a
+    # process execution. This wouuld be bad, as the results can be big and
+    # cause memory to fragment.
+    OK_TO_INSTANTIATE = False
```

Comment:
You are indeed correct. I thought that the convention was to make class variables upper case, but I was wrong. Fixing.